src/apps/fuzzers/fuzzerGridDisk.c

```diff
@@ -58,6 +58,11 @@ int LLVMFuzzerTestOneInput(const uint8_t *data, size_t size) {
     free(results);
 
     // TODO: use int64_t
+    results = calloc(sizeof(H3Index), sz);
+if (results != NULL) {
+    H3_EXPORT(gridRing)(args->index, args->k, results);  // Test the safe version
+}
+free(results);
```

Comment:
```suggestion
    if (results != NULL) {
        H3_EXPORT(gridRing)(args->index, args->k, results);  // Test the safe version
    }
    free(results);
```
inadvertent formatting issue?